filament/backend/src/vulkan/VulkanHandles.cpp

```diff
@@ -382,12 +264,15 @@ VulkanTexture::VulkanTexture(VulkanContext& context, SamplerType target, uint8_t
         .format = vkformat,
         .mipLevels = levels,
         .arrayLayers = 1,
-        .usage = VK_IMAGE_USAGE_SAMPLED_BIT,
+        .usage = 0,
         .samples = VK_SAMPLE_COUNT_1_BIT,
     };
+    if (usage & TextureUsage::SAMPLEABLE) {
+        imageInfo.flags |= VK_IMAGE_USAGE_SAMPLED_BIT;
```

Comment:
USAGE_SAMPLED moved from being set on .usage to .flags, is that intentional?

Reply:
Wow thanks for the catch.  Amended a fix.